device_tracker.py

```diff
@@ -54,6 +54,7 @@ def get_scanner(hass, config):
     return KismetDeviceScanner(hass, config[DOMAIN])
 
 
+DeviceGPS = namedtuple('Device', ['mac', 'last_ssid', 'gps', 'last_update'])
 Device = namedtuple('Device', ['mac', 'last_ssid', 'last_update'])
```

Comment:
I'm not sure if there is a cleaner way to do this, but I suspect that the newer code won't need this

Reply:
But, could just use Device with GPS and send (0,0) ...